src/ontology/uberon-edit.obo

```diff
@@ -50856,6 +50856,7 @@ synonym: "lateral hypothalamic group" EXACT []
 synonym: "lateral hypothalamic nucleus" RELATED [MA:0000834]
 synonym: "lateral hypothalamic region" EXACT [FMA:62030]
 synonym: "lateral hypothalamic zone (Crosby)" EXACT []
+synonym: "lateral hypothalamus" EXACT []
 synonym: "LH" BROAD ABBREVIATION [BIRNLEX:4037, NIFSTD:NeuroNames_abbrevSource]
```

Comment:
@cmungall, the conflict in the branch is related to the `ABBREVIATION` synonym type that was changed to `OMO:0003000` a long time ago. I'm not sure it didn't show in the diff. Have you created the branch from an updated version of the master?